lib/mozart_fetcher/component.ex

```diff
@@ -42,7 +42,7 @@ defmodule MozartFetcher.Component do
   end
 
   defp get(config) do
-    LocalCache.get_or_store(config.endpoint, fn -> HTTPClient.get(config.endpoint) end)
+    LocalCache.get_or_store(config.endpoint, fn -> ComponentFetcher.fetch(config.endpoint) end)
```

Comment:
Naming for this can be decided/changed at later date. This could be ComponentParser or ResponseHandler.

@ettomatic may have some thoughts on this.